src/styles/globals.css

```
@@ -3,13 +3,13 @@
 @tailwind utilities;
 
 body {
-    font-family: '--font-geist-sans', sans-serif;
-    background-color: #2c2f33 
+  font-family: "--font-geist-sans", sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "--font-geist-sans" (font-family-name-quotes)

(font-family-name-quotes)